Not able to install zeromq on OsX Macbook Air M2

Not able to install zeromq using npm install zeromq. It fails with this error:

npm ERR! command failed
npm ERR! signal SIGINT
npm ERR! command sh -c (npm run build.js || echo ok) && aminya-node-gyp-build --build-from-source
npm ERR! > zeromq@6.0.0 build.js
npm ERR! > run-p build.script build.library
npm ERR! 
npm ERR! 
npm ERR! > zeromq@6.0.0 build.library
npm ERR! > tsc -p ./src/tsconfig.json
npm ERR! 
npm ERR! 
npm ERR! > zeromq@6.0.0 build.script
npm ERR! > tsc -p ./script/tsconfig.esm.json && tsc -p ./script/tsconfig.json
npm ERR! 
npm ERR! error TS2688: Cannot find type definition file for 'mocha'.
npm ERR!   The file is in the program because:
npm ERR!     Entry point of type library 'mocha' specified in compilerOptions
npm ERR! error TS2688: Cannot find type definition file for 'node'.
npm ERR!   The file is in the program because:
npm ERR!     Entry point of type library 'node' specified in compilerOptions
npm ERR! ok
npm ERR! ERROR: "build.script" exited with 2.

I'm running Mac OS 14.2.1 (23C71)

--- a/package.json
+++ b/package.json
@@ -82,7 +82,7 @@
     "tsconfig.json"
   ],
   "scripts": {
-    "install": "(npm run build.js || echo ok) && aminya-node-gyp-build --build-from-source",
+    "install": "(npm run build.js || echo ok) && cross-env npm_config_build_from_source=true aminya-node-gyp-build",
     "clean": "shx rm -rf ./build ./lib/ ./prebuilds ./script/*.js ./script/*.mjs ./script/*.js.map ./script/*.mjs.map ./script/*.d.ts ./script/*.d.mts ./script/*.cjs ./scripts/*.cjs.map ./scripts/*.d.cts ./script/*.tsbuildinfo",
     "clean.release": "shx rm -rf ./build/Release",
     "clean.temp": "shx rm -rf ./tmp && shx mkdir -p ./tmp",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -6,10 +6,7 @@
     "declaration": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "types": [
-      "node",
-      "mocha"
-    ],
+    "forceConsistentCasingInFileNames": true,
     "strictPropertyInitialization": false, // TODO
     "strict": true,
     "strictNullChecks": true,

